.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - '4'
```

Comment:
The reorder because for most cases the newer the version the faster the job completes

Dropping 7.x because 8.x supersedes 7.x, there's no LTS planned for 7.x

See https://github.com/nodejs/LTS#lts-schedule1